google/cloud/forseti/common/gcp_api/errors.py

```diff
@@ -25,16 +40,16 @@ class ApiExecutionError(Error):
     CUSTOM_ERROR_MESSAGE = (
         'GCP API Error: unable to get {0} from GCP:\n{1}\n{2}')
 
-    def __init__(self, resource_name, e):
+    def __init__(self, api_execution_arg, e):
```

Comment:
If you are up for it, I really don't like this being a required arg. It's messy and used inconsistently across 2.0. I would prefer to rip it out and remove everyplace it's used in a raise call.

Reply:
updated the constructor to take in more arguments w/ default values to avoid the issue you mentioned